form/src/main/java/org/dhis2/form/ui/FormViewModel.kt

```diff
@@ -173,7 +173,10 @@ class FormViewModel(
     ): Throwable? {
         return fieldValue?.let { value ->
             var error =
-                when (val result = valueType?.validator?.validate(value)) {
+                when (
+                    val result = valueType?.takeIf { it != ValueType.IMAGE }
```

Comment:
ValueType.IMAGE has the following validator:
` IMAGE(String::class.java, UidValidator);`
Is this class: UidValidator.kt. Why can not be validated?

Reply:
The problem is when reaching this validator the value of image field is not a UID, is the path of the file that has to be stored. Maybe we could discuss if this validator could be skipped or we should make something else.